### DISCUSSION TITLE
Auto update

### DISCUSSION THREAD

**Discussion by Nicolol1234** [PurpurMC/Purpur#1398] [__Dismissed__] [answered]
Add a command to automatically download a new update of purpur. Like /purpur update or something, because my internet is so bad it take an eternity to update

**Comment by BillyGalbreath** [comment, marked as answer]
![image](https://github.com/PurpurMC/Purpur/assets/332527/29a82fab-f6c3-4a45-a827-9ab2c9e764b8)
